packages/datadog_tracking_http_client/.flutter-plugins

```diff
@@ -0,0 +1,2 @@
+# This is a generated file; do not edit or check into version control.
```

Comment:
seems it was checked into version control :)

Reply:
🤔 Why isn't it in the default .gitignore I wonder...